src/components/MultiLink/Legend.vue

```diff
@@ -28,8 +31,8 @@ export default Vue.extend({
 
   data() {
     return {
-      svgHeight: 150,
-      yAxisPadding: 10,
+      svgHeight: 50,
+      yAxisPadding: 25, // Gives enough width for hundreds on the y axis
```

Comment:
Obviously, this will run into issues if you ever want to plot thousands, etc. We will eventually want a better way to decide how much padding to use; that's something @jtomeck might be able to help with.

Reply:
I figured hundreds would be a safe value since for thousands, we'd need to load at least 1000 nodes, which is currently too much for the vis.

Reply:
Gotcha. Well, if it ever does scale that high, we can fix this then.